setup.py

```diff
@@ -23,12 +48,8 @@
     license='MIT license',
     packages=('winrm', 'winrm.tests', 'winrm.vendor.requests_kerberos'),
     package_data={'winrm.tests': ['*.ps1']},
-    install_requires=['xmltodict', 'requests>=2.9.1', 'requests_ntlm>=0.3.0', 'six'],
-    extras_require={
-        'credssp': ['requests-credssp>=1.0.0'],
-        'kerberos:sys_platform=="win32"': ['winkerberos>=0.5.0'],
-        'kerberos:sys_platform!="win32"': ['pykerberos>=1.2.1,<2.0.0']
-    },
+    install_requires=req_deps_list,
```

Comment:
don't do this, tox can use the extras and install_requires, just make sure to set `skipsdist = False

Reply:
then you can do:
```
[testenv]
extras = test,credssp,kerberos
```